CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Additions and Improvements
 * Added `besu_transaction_pool_transactions` to the reported metrics, counting the mempool size [\#1869](https://github.com/hyperledger/besu/pull/1869)
+* Distributions and maven artifacts have been moved off of bintray [\#1886](https://github.com/hyperledger/besu/pull/1886)
```

Comment:
would it be more accurate to say "also publishing to artifactory" since we are still doing the bintray upload for now

Reply:
We will only be able to do dual publishing for this release only, I'm only keeping it in as a hedge and plan to delete it as the first commit if all goes well.